packages/gestalt/src/Table.js

```diff
@@ -10,23 +11,61 @@ import TableHeaderCell from './TableHeaderCell.js';
 import TableRowExpandable from './TableRowExpandable.js';
 import TableRow from './TableRow.js';
 import TableSortableHeaderCell from './TableSortableHeaderCell.js';
+import TableContext from './TableContextProvider.js';
 
 type Props = {|
   children: Node,
   borderStyle?: 'sm' | 'none',
   maxHeight?: number | string,
+  stickyColumns?: number,
 |};
 
 export default function Table(props: Props): Node {
-  const { borderStyle, children, maxHeight } = props;
+  const { borderStyle, children, maxHeight, stickyColumns } = props;
+  const [showShadowScroll, setShowShadowScroll] = useState(null);
```

Comment:
I don't think Flow will be able to properly infer the type here, so you may want to add it. Looks like it's an optional enum? `?('left' | 'right')`

Reply:
![image](https://user-images.githubusercontent.com/10593890/113351249-97c19600-9308-11eb-80e7-3746f2ee4197.png)

Reply:
nit, I think ryan was suggesting here `useState<'left' | 'right'>()`